src/sourcemaps/utils/sdk-version.ts

```diff
@@ -0,0 +1,213 @@
+ *    -> We tell them to install an SDK and then continue with the wizard
+ * 2. Users installed an SDK in the range >=7.47.0
+ *    -> All good, no need to do anything!
+ * 3. Users installed an SDK in the range >=7.0.0 <= 7.46.0
+ *    -> We ask if they want to auto-update to the latest version
+ * 4. Users installed an SDK in the range <7.x
+ *    -> We tell users to manually upgrade (migrate between majors)
+ */
+export async function ensureMinimumSdkVersionIsInstalled(): Promise<void> {
+  const packageJson = await getPackageDotJson();
+
+  const installedSdkPackage = SENTRY_SDK_PACKAGES.find((sdkPackage) =>
+    hasPackageInstalled(sdkPackage, packageJson),
+  );
+
+  // Case 1:
+  if (!installedSdkPackage) {
+    return await handleNoSdkInstalled();
+  }
+
+  const installedSdkVersionOrRange =
+    getPackageVersion(installedSdkPackage, packageJson) || '';
+  const minInstalledVersion =
+    minVersion(installedSdkVersionOrRange)?.version || '';
+
+  const hasDebugIdCompatibleSdkVersion = satisfies(
+    minInstalledVersion,
+    `>=${MINIMUM_DEBUG_ID_SDK_VERSION}`,
+  );
+
+  // Case 2:
+  if (hasDebugIdCompatibleSdkVersion) {
+    Sentry.setTag('initial_sdk_version', '>=7.47.0');
+    return;
+  }
+
+  const hasV7SdkVersion = satisfies(minInstalledVersion, '>=7.0.0');
+
+  clack.log.warn(
+    `${chalk.yellowBright(
+      `It seems like you're using an outdated version (${installedSdkVersionOrRange}) of the ${chalk.bold(
+        installedSdkPackage,
+      )} SDK.`,
+    )}
+Uploading source maps is easiest with an SDK from version ${chalk.bold(
+      MINIMUM_DEBUG_ID_SDK_VERSION,
+    )} or newer.    
+`,
+  );
+
+  // Case 3:
+  if (hasV7SdkVersion) {
+    await handleAutoUpdateSdk(installedSdkPackage);
+    return;
+  }
+
+  // Case 4:
+  await handleManuallyUpdateSdk(minInstalledVersion);
+}
+
+async function handleManuallyUpdateSdk(minInstalledVersion: string) {
+  Sentry.setTag(
+    'initial_sdk_version',
+    `${satisfies(minInstalledVersion, '>=6.0.0') ? '6.x' : '<6.0.0'}`,
+  );
+
+  clack.log
+    .info(`When upgrading from a version older than 7.0.0, make sure to follow the migration guide:
+https://github.com/getsentry/sentry-javascript/blob/develop/MIGRATION.md#upgrading-from-6x-to-7x
+`);
+
+  const didUpdate = await abortIfCancelled(
+    clack.select({
+      message: 'Did you update your SDK to the latest version?',
+      options: [
+        {
+          label: 'Yes!',
+          value: true,
+        },
+        {
+          label: "No, I'll do it later...",
+          value: false,
+          hint: chalk.yellow(
+            `Remember to update your SDK to at least ${MINIMUM_DEBUG_ID_SDK_VERSION}.`,
+          ),
+        },
+      ],
+      initialValue: true,
+    }),
+  );
+
+  Sentry.setTag(
+    'resolved_sdk_status',
+    didUpdate ? 'updated_manually' : 'update_later',
+  );
+}
+
+async function handleAutoUpdateSdk(packageName: string) {
+  Sentry.setTag('initial_sdk_version', '>=7.0.0 <= 7.47.0');
```

Comment:
```suggestion
  Sentry.setTag('initial-sdk-version', '>=7.0.0 <= 7.47.0');
```

giga l: All of our tags so far are kebab case